### ISSUE TITLE
Add file information to error messages

### ISSUE THREAD

**Issue by vaadin-marcus** [Polymer/polymer-cli#268]
I'm building a bundled project with `polymer build` on 0.11.1 and it fails with the following error:

```
error:   Uncaught exception: TypeError: Cannot read property 'is' of null
error:   TypeError: Cannot read property 'is' of null
    at Parser.parseDeclarationOrRuleset (/Users/mhellber/.nvm/versions/node/v6.0.0/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:191:33)
    at Parser.parseRule (/Users/mhellber/.nvm/versions/node/v6.0.0/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:80:21)
    at Parser.parseRules (/Users/mhellber/.nvm/versions/node/v6.0.0/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:62:25)
    at Parser.parseStylesheet (/Users/mhellber/.nvm/versions/node/v6.0.0/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:55:47)
    at Parser.parse (/Users/mhellber/.nvm/versions/node/v6.0.0/lib/node_modules/polymer-cli/node_modules/shady-css-parser/dist/shady-css/parser.js:51:19)
    at css (/Users/mhellber/.nvm/versions/node/v6.0.0/lib/node_modules/polymer-cli/node_modules/css-slam/index.js:51:39)
    at GulpTransform._transform (/Users/mhellber/.nvm/versions/node/v6.0.0/lib/node_modules/polymer-cli/node_modules/css-slam/index.js:69:36)
    at GulpTransform.Transform._read (_stream_transform.js:167:10)
    at GulpTransform.Transform._write (_stream_transform.js:155:12)
    at doWrite (_stream_writable.js:306:12)
    at writeOrBuffer (_stream_writable.js:292:5)
```

It would be very helpful for debugging if the error message could contain information on the file that failed similar to warning messages. I'm sure the actual error is in my code or it's dependencies, but it would be very helpful to get more debug info.

**Comment by justinfagnani** [comment]
By any chance do you have a repro for this so we can make sure we get the file info here?

**Comment by vaadin-marcus** [comment]
Here you go, just run `polymer build` on this. 

I narrowed down the issue in this particular case to an incorrect stylesheet in our `vaadin-grid` demo folder (`bower_components/vaadin-grid/demo/common.css`). The dependency version is set to a version that still has the issue present. 

I was hoping we could come up with a way to show file info (and maybe even line) more generally when errors occur if that's possible.

Thanks for looking into this.

**Comment by justinfagnani** [comment]
I just dug in, and one problem here is that some of these errors are occurring in support libraries, which themselves are doing file crawling. There's no change we can make in CLI to fix this particular one, Hydrolysis needs to be updated. We might want to turn this into a tracking bug for each error that doesn't have associated file info.

**Comment by FredKSchott** [comment]
Fixed in [v0.13.0](https://github.com/Polymer/polymer-cli/releases/tag/v0.13.0)
